subscriptions/management/commands/fix_subscription_lifecycle.py

```diff
@@ -30,7 +30,8 @@ def handle(self, *args, **options):
 
         updated = 0
 
-        subscriptions = Subscription.objects.filter(active=True)
+        subscriptions = Subscription.objects.filter(active=True)\
+            .exclude(process_status=-1)
```

Comment:
Would we maybe want to filter by `process_status=0` instead? Since -1 and -2 are error states, 1 is that is busy processing, and 2 is that the subscription has reached the end of the message set and has ended. So I don't see a need for this command to work on anything except for 0?